stream.go

```diff
@@ -1,7 +1,9 @@
 package compress
 
 import (
-	"bytes"
+	bytesLib "bytes"
```

Comment:
stylistic comment; I'ld rather have this guy still named `bytes` and the variable in `FillBytes` be `data` or something else